### DISCUSSION TITLE
Wondering about values on the rundown endpoint

### DISCUSSION THREAD

**Discussion by SirArchibald97** [Noxcrew/mcchampionship-api#7] [Q&A] [answered]
So I am using some endpoints to get some data for a project, and I may have noticed a discrepancy in the values it fetches from `https://api.mcchampionship.com/v1/rundown`, unless I am being stupid. It seems the total team scores (`data.eventScores`) are far higher than the individual scores (`data.individualScores`) added together, so I am wondering if there is any way these scores can be inflated so massively or are they are just wrong?

**Comment by MagicA550** [comment, marked as answer]
Hello,

Event scores are inflated as they account for the coin multiplier, whereas individual scores are not. This is why individual scores do not sum to the total event score for each team.

There is a sizable feature in the pipeline which will hopefully give better breakdowns soon; this issue is simply due to the way in which the event server displays these scores (and hence how it hands them off to MCC Live).

Hope this helps!

**Reply by SirArchibald97** [reply]
I guessed it was something like that, thanks for the reply! Looking forward to seeing what else you guys put out!
